lib/screen/profile_screen.dart

```diff
@@ -68,30 +69,70 @@ class ProfileScreen extends StatelessWidget {
                             onPressed: () => {},
                             toggle: false,
                           ),
-                          const SizedBox(
-                            width: 25,
-                          ),
-                          TunedInOutlinedButton(
-                            text: "Join",
-                            onPressed: () => {},
-                            toggle: true,
+                          SizedBox(
+                            width: group ? 25 : 0,
                           ),
+                          group
+                              ? TunedInOutlinedButton(
+                                  text: "Join",
+                                  onPressed: () => {},
+                                  toggle: true,
+                                )
+                              : const SizedBox.shrink(),
                         ],
                       )),
                   const SizedBox(height: 10),
                   Row(
                     mainAxisAlignment: MainAxisAlignment.center,
-                    children: [
-                      Text(
-                        "200",
-                        style:
-                            Theme.of(context).textTheme.displaySmall?.copyWith(
-                                  fontWeight: FontWeight.w600,
-                                ),
-                      ),
-                      Text(" Followers",
-                          style: Theme.of(context).textTheme.displaySmall)
-                    ],
+                    children: !group
+                        ? [
+                            // Text(
+                            //   "200",
+                            //   style:
+                            //       Theme.of(context).textTheme.displaySmall?.copyWith(
+                            //             fontWeight: FontWeight.w600,
+                            //           ),
+                            // ),
+                            // Text(" Followers",
+                            //     style: Theme.of(context).textTheme.displaySmall),
+                            // const SizedBox(width: 25, height: 2),
+                            // Text(
+                            //   "200",
+                            //   style:
+                            //       Theme.of(context).textTheme.displaySmall?.copyWith(
+                            //             fontWeight: FontWeight.w600,
+                            //           ),
+                            // ),
+                            // Text(" Following",
+                            //     style: Theme.of(context).textTheme.displaySmall)
+
+                            const SizedBox(
+                              width: 100,
+                              child: Text("Followers\n 200",
```

Comment:
You should probably go off of what used for the text widgets below for when it is a group for styling. In addition, split the label and follower count between 2 different widgets (unified by a column widget) - this is the preferable way to do it in Flutter so that we can style both differently (as seen in the Figma design) and also apply localization better (if we need it down the road)